src/imitation/data/wrappers.py

```diff
@@ -235,30 +234,8 @@ def __init__(self, env: Env, original_obs_key: str = "ORI_OBS"):
             )
         self._original_obs_key = original_obs_key
         super().__init__(env)
-        self._update_obs_space()
```

Comment:
I think we still need the observation space update. From the [Gynmasium docs](https://gymnasium.farama.org/api/wrappers/observation_wrappers/):
> The transformation defined in that method must be reflected by the env observation space. Otherwise, you need to specify the new observation space of the wrapper by setting self.observation_space in the __init__() method of your wrapper.